CRITICAL - 2022-03-01 22:18:48 --> Argument 2 passed to view() must be of the type array, null given, called in C:\xampp\htdocs\ci4\app\Controllers\News.php on line 35
#0 C:\xampp\htdocs\ci4\app\Controllers\News.php(35): view('news/welcome', NULL)
#1 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->login()
#2 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-01 22:19:51 --> Argument 2 passed to view() must be of the type array, null given, called in C:\xampp\htdocs\ci4\app\Controllers\News.php on line 35
#0 C:\xampp\htdocs\ci4\app\Controllers\News.php(35): view('news/welcome', NULL)
#1 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->login()
#2 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-01 22:19:54 --> Argument 2 passed to view() must be of the type array, null given, called in C:\xampp\htdocs\ci4\app\Controllers\News.php on line 35
#0 C:\xampp\htdocs\ci4\app\Controllers\News.php(35): view('news/welcome', NULL)
#1 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->login()
#2 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2022-03-01 22:44:46 --> Invalid file: http://localhost/ci4/public/news/error.php
#0 C:\xampp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('http://localhos...')
#1 C:\xampp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('http://localhos...', Array, true)
#2 C:\xampp\htdocs\ci4\app\Controllers\News.php(189): view('http://localhos...')
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->login()
#4 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#5 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-01 22:46:20 --> Invalid file: ci4/public/news/error.php
#0 C:\xampp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('ci4/public/news...')
#1 C:\xampp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('ci4/public/news...', Array, true)
#2 C:\xampp\htdocs\ci4\app\Controllers\News.php(189): view('ci4/public/news...')
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->login()
#4 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#5 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-01 22:46:31 --> Invalid file: ci4/public/news/error.php
#0 C:\xampp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('ci4/public/news...')
#1 C:\xampp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('ci4/public/news...', Array, true)
#2 C:\xampp\htdocs\ci4\app\Controllers\News.php(189): view('ci4/public/news...')
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->login()
#4 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#5 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-01 22:51:49 --> Invalid file: news/welome.php
#0 C:\xampp\htdocs\ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('news/welome.php')
#1 C:\xampp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('news/welome', Array, true)
#2 C:\xampp\htdocs\ci4\app\Controllers\News.php(186): view('news/welome')
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->login()
#4 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#5 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-01 23:18:49 --> syntax error, unexpected '<'
#0 C:\xampp\htdocs\ci4\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 C:\xampp\htdocs\ci4\system\Common.php(1089): CodeIgniter\View\View->render('templates/heade...', Array, true)
#2 C:\xampp\htdocs\ci4\app\Controllers\News.php(19): view('templates/heade...')
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->index()
#4 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#5 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-03-01 23:25:21 --> Call to undefined method CodeIgniter\Session\Session::unset()
#0 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->logout()
#1 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#2 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-03-01 23:39:42 --> Call to a member function destroy() on null
#0 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->logout()
#1 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#2 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#3 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#4 {main}
CRITICAL - 2022-03-01 23:40:08 --> syntax error, unexpected '->' (T_OBJECT_OPERATOR)
#0 C:\xampp\htdocs\ci4\system\Autoloader\Autoloader.php(241): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\ci4\system\Autoloader\Autoloader.php(219): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\ci4\system\CodeIgniter.php(790): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\ci4\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2022-03-01 23:41:35 --> session_destroy(): Trying to destroy uninitialized session
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'session_destroy...', 'C:\\xampp\\htdocs...', 208, Array)
#1 C:\xampp\htdocs\ci4\app\Controllers\News.php(208): session_destroy()
#2 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->logout()
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#4 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2022-03-01 23:41:39 --> session_destroy(): Trying to destroy uninitialized session
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'session_destroy...', 'C:\\xampp\\htdocs...', 208, Array)
#1 C:\xampp\htdocs\ci4\app\Controllers\News.php(208): session_destroy()
#2 C:\xampp\htdocs\ci4\system\CodeIgniter.php(825): App\Controllers\News->logout()
#3 C:\xampp\htdocs\ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\News))
#4 C:\xampp\htdocs\ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
